callback.js

```diff
@@ -0,0 +1,119 @@
+////////////////##Question One
+function isCaller(callback) {
+    callback()
+}
+
+function sayHello() {
+    console.log("Hello!")
+}
+isCaller(sayHello);
+
+
+////////////////##Question Two
+const ignoreCallback = (callback, boolean) => {
+    if (boolean === true) {
+        ignoreCallback();
+    } else {
+        console.log("Ignore the callback")
+    }
+}
```

Comment:
function not returning. It's calling itself on boolean. It should be calling the callback.